content/en/docs/concepts/configuration/configmap.md

```diff
@@ -216,26 +216,6 @@ data has the following advantages:
 - improves performance of your cluster by significantly reducing load on kube-apiserver, by
   closing watches for ConfigMaps marked as immutable.
 
-This feature is controlled by the `ImmutableEphemeralVolumes`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
-You can create an immutable ConfigMap by setting the `immutable` field to `true`.
-For example:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  ...
-data:
-  ...
-immutable: true
-```
-
-Once a ConfigMap is marked as immutable, it is _not_ possible to revert this change
-nor to mutate the contents of the `data` or the `binaryData` field. You can
-only delete and recreate the ConfigMap. Because existing Pods maintain a mount point
-to the deleted ConfigMap, it is recommended to recreate these pods.
```

Comment:
Please leave these lines; they document the feature.

Reply:
sure, just remove the feature gate part.